.gitignore

```diff
@@ -16,3 +16,8 @@ modules/ROOT/.DS_Store
 modules/ROOT/.DS_Store
 modules/ROOT/.DS_Store
 modules/ROOT/.DS_Store
+modules/ROOT/.DS_Store
```

Comment:
Just one of these is needed.

Reply:
This has to be the most helpful comment that I've ever received during a review. Thanks for your attention to detail.